web/impact/impact/v1/classes/option_analysis.py

```diff
@@ -158,13 +161,19 @@ def calc_capacity(self, option_name, helper, spec_helper):
             "remaining_capacity": remaining_capacity,
         }
 
-    def populate_criterion_total_capacities_cache(self, field, option_name):
-        '''Query total capacity data and cache it.'''
+    def populate_criterion_total_capacities_cache(self,
+                                                  field,
+                                                  option_name,
+                                                  cache_key):
+        '''Query total capacity data and cache it.
+        Use annotation to ensure that cache keys are consistent'''
         if self.criterion_total_capacities.get(option_name) is None:
-            capacities = JudgeRoundCommitment.objects.filter(
-                judging_round=self.judging_round).values(field).annotate(
-                    total=Sum("capacity"))
-            result = {cap[field]: cap["total"]
+            capacities = JudgeRoundCommitment.objects.annotate(
+                **{cache_key: F(field)}).filter(
```

Comment:
Can we do this query without using `**`? It impairs readability for sure. If I have to fix a bug involving these lines, I will have to really slow down to figure out what this means.

If for some reason it must be used, it would be a kindness to include a brief comment  explaining how the query works.